pkg/registry/apis/provisioning/apifmt/error.go

```diff
@@ -0,0 +1,109 @@
+// apifmt aims to provide a Kubernetes-compatible way to format text.
```

Comment:
can you elaborate on this one, please?

Reply:
Kubernetes does not like `fmt.Errorf` calls: it will see them as an internal server error no matter what it actually contains. In most instances, we construct the apierror such that we can propagate the status upwards, but it gets lost.